webapp/crashstats/libbugzilla.py

```diff
@@ -50,12 +51,10 @@ def frame_generator(thread):
     for frame in islice(frame_generator(thread), MAX_FRAMES):
         # Source is an empty string if data isn't available
         source = frame.get("file") or ""
-        if frame.get("line"):
-            source += ":{}".format(frame["line"])
+        if source and frame.get("line") is not None:
+            source = f"{source}:{frame['line']}"
```

Comment:
I redid how these get constructed to ditch the `.format()` call and also fix the situation where there isn't a file, but there is a line. Previously, that would result with a source of something like `:20` which is weird. Twentieth line of what exactly?

Does that happen in the wild? I have no idea and no good way to hunt for an instance. But it is what the previous implementation did.

Reply:
is the change from `frame.get("line")` to `frame.get("line") is not None` because line can be 0?

Reply:
Yes!